### ISSUE TITLE
Beeg crashing

### ISSUE THREAD

**Issue by falbani** [dobbelina/repository.dobbelina#405]
The
`video["duration"]`
contains some null values
`"duration":null,`

and when trying to do
`m, s = divmod(video["duration"], 60)`
it will crash saying
_TypeError: int() argument must be a string or a number, not 'NoneType'_

I made a quick fix doing this, but I am sure you guys will find a better way
```
if video["duration"] is not None:
    m, s = divmod(video["duration"], 60)
else:
    m, s = divmod(0, 60)
```

thank you for this AWESOME plugin!

**Comment by Gujal00** [comment]
Which addon? it works fine in cumination

**Comment by falbani** [comment]
> Which addon? it works fine in cumination
Are you sure? Maybe you didn't get any null value from beeg, try again it should crash eventually...

I got crashes in two machines 
Uwc @ Kodi 1.76 krypton @ Windows 10
Cumination @ Kodi 1.8 leia @ Android TV

**Comment by 12asdfg12** [comment]
Fixed in cumination 1.1.3
